### PR TITLE
Fixed Typo's and grammar 

### PR DESCRIPTION
**Pull request by vannut**
Some minor typo's and grammar enhancements for the dutch language file. @CptMeatball

### CODE REVIEW

File: lang/nl/lang.php [hunk 1]
```diff
@@ -43,7 +43,7 @@
         'scan_messages_success' => 'De thema bestanden zijn succesvol gescand!',
         'scan_messages_hint' => 'Klikken op <strong>Zoeken naar nieuwe berichten</strong> controleert het huidige thema bestanden voor nieuwe berichten om te vertalen. ',
```
Comment (vannut) [inline lang/nl/lang.php]: This is not a good sentence.

**Comment by vannut** [comment]
Damn you're fast @LukeTowers :)

File: lang/nl/lang.php (hunk 1 above)
Comment (CptMeatball) [reply lang/nl/lang.php]: Yes sounds better!

**Comment by CptMeatball** [comment]
Aii, should be more careful next time with those translations.

Kudos @vannut !

**Comment by LukeTowers** [comment]
Faster merger in the west @vannut ;)

File: lang/nl/lang.php (hunk 1 above)
Comment (vannut) [reply lang/nl/lang.php]: I'll produce another PR for it :) @LukeTowers is to fast :)

**Pull request merged**